### PR TITLE
Assemble 1.21.40

### PR DESCRIPTION
**Pull request by dries-c**
## Introduction
This PR updates PM5 to 1.21.40

## Backwards compatibility
See https://github.com/pmmp/BedrockProtocol/pull/277

### CODE REVIEW

**Review by dktapps — CHANGES_REQUESTED** [review]
Looks like the mushroom_stem changes aren't accounted for

File: src/network/mcpe/handler/PreSpawnPacketHandler.php
Comment (dktapps) [inline src/network/mcpe/handler/PreSpawnPacketHandler.php:101]: Why is this changed to v2?

Comment (cooldogedev) [reply src/network/mcpe/handler/PreSpawnPacketHandler.php:101]: It is correct, the old [one](https://github.com/pmmp/BedrockProtocol/blob/9825e3abdd43f13f54aa359cce5020c5e303541c/src/types/PlayerMovementType.php#L20)'s value was `1` and so is the [new](https://github.com/pmmp/BedrockProtocol/blob/6aa7cbeb4a7ec6fa58f9024aeaddad7c5c65a459/src/types/ServerAuthMovementMode.php#L21)

Comment (dktapps) [reply src/network/mcpe/handler/PreSpawnPacketHandler.php:101]: Thanks

**Review by DevPapo — COMMENTED** [review]
Ok

**Comment by joinasau** [comment]
...
merge?

**Review by ShockedPlot7560 — DISMISSED** [review]

**Comment by AzaleeX** [comment]
> ... merge?

We need a second approval before we can merge, this is a security feature.

**Comment by smell-of-curry** [comment]
looks good?

**Comment by AzaleeX** [comment]
> looks good?

Yes, I think it is, but there is no one to do the review, so it takes time, I think.

**Comment by dktapps** [comment]
There are still unaddressed review comments

**Comment by AzaleeX** [comment]
> There are still unaddressed review comments

What else does he have to change?

**Review by dktapps — APPROVED** [review]

**Pull request merged**
